### ISSUE TITLE
Upgrade to Jekyll 2.5.x

### ISSUE THREAD

**Issue by jnweaver** [UWMadisonUcomm/uw-ui-toolkit#66]

**Comment by ethan-nelson** [comment]
:+1: collections support would be very useful.

**Comment by jnweaver** [comment]
Interesting: I didn't even know Jekyll supported such a thing. I have not kept up with Jekyll too much in the last year or more.

We're mainly using Jekyll just to support the website for this project and not necessarily as a platform for others to publish UW content in. But there's of course no reasons someone could not use it that way too. Maybe we'll add collections support. Though it's marked as unstable right now.

Anyway, thanks for the suggestion.

**Comment by ethan-nelson** [comment]
Fair enough :) I guess I used this as a good skeleton start since the themes are already configured, but I am using my own version of Jekyll at the moment.

If this package does ever get adapted for wider use, collections provide a great way for fast page population. In example, I have a _faculty folder containing files for each member that has mostly front matter with all the contact information and details. Then, I can have a main faculty page listing them all with the information populated through Jekyll by calling `site.faculty`. Working on a similar setup for event pages, too.
